### ISSUE TITLE
Problem with non-ASCII characters (ñ) (á) (Á) etc

### ISSUE THREAD

**Issue by Alvy** [kochrt/timeline-maker#118]
There seems to be a problem with non-ASCII characters in languages other than English.

I can't edit text with letters such as the Spanish (ñ) (Ñ) or accented letters (á) (é) …

This appears to be an issue not only in the browser but also when using copy and paste from a text editor.

Additionally, linked text and tags like "#diseño" get split into something like "#dise // ño // ño," repeating after the (ñ) letter.

**Comment by kochrt** [comment]
Yeah it looks like the [regex for tags](https://github.com/mark-when/parser/blob/main/src/regex.ts#L246) doesn't play well with those characters. Adding the [`u`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_expressions/Unicode_property_escapes) flag should help. 


Aside from tags, what were there other issues you were noticing regarding non-ascii/unicode characters? I'll need to figure out which regexes need the flag

**Comment by Alvy** [comment]
It seems to be working fine now.

- #Tags are okay.
- [Text in brackets for links is okay.]
- Text outside of links is also okay.

So, great. Thank you, and keep up the good work!

**Comment by kochrt** [comment]
I haven't implemented any fixes yet - I'm glad it's working for you, but I just tried and it's still broken for me 😂

**Comment by Alvy** [comment]
Pretty strange; I'm using the same browser on the same file.

Anyway, the three problemes were #tags, [text in brackets] and sometimes text outside of brackets also.

I'll let you know if I notice anything again. (BTW I'm using Brave on Mac OS X.)
